src/i18NextInit.js

```diff
@@ -15,12 +17,16 @@ i18n
     },
     nsSeparator: false, // namespace separator
     keySeparator: false, // key separator
-    defaultNs: 'translation',
+    ns: ['translations'],
+    defaultNS: 'translations',
     fallbackNS: 'translation',
```

Comment:
Do we want that `fallbackNS` to be `'translation'` if namespaces changed to `'translations'`?